src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -1,7 +1,9 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 public class Task3 {
-    public static boolean isBetween(int a, int b, int c) {
-        return false;
+    public static boolean isBetween(int a, int b, int c)
+    {
+        return
+                a<c? b>=a && b<=c? true: false: b<=a && b>=c? true: false; // Test 3 done.
```

Comment:
It's extremely difficult to read such code. Use '(' and ')' to improve readability. Moreover 
`b >= a && b <= c ? true : false` and `b <= a && b >= c ? true : false` can be simplified. As you may aware result of logic operations is boolean, so there is no need to use ternary operator.